aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -104,27 +104,95 @@ To define a JS initializer, add a JS module to the project named `{NAME}.lib.mod
+* `afterStarted(blazor)`: Called after Blazor is ready to receive calls from JS. For example, `afterStarted` is used to initialize libraries by making JS interop calls and registering custom elements. The Blazor instance is passed to `afterStarted` as an argument (`blazor`).
 
 For the file name:
 
 * If the JS initializers are consumed as a static asset in the project, use the format `{ASSEMBLY NAME}.lib.module.js`, where the `{ASSEMBLY NAME}` placeholder is the app's assembly name. For example, name the file `BlazorSample.lib.module.js` for a project with an assembly name of `BlazorSample`. Place the file in the app's `wwwroot` folder.
 * If the JS initializers are consumed from an RCL, use the format `{LIBRARY NAME/PACKAGE ID}.lib.module.js`, where the `{LIBRARY NAME/PACKAGE ID}` placeholder is the project's library name or package identifier. For example, name the file `RazorClassLibrary1.lib.module.js` for an RCL with a package identifier of `RazorClassLibrary1`. Place the file in the library's `wwwroot` folder.
 
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+<!-- UPDATE 8.0 Remove NOTE -->
+
+> [!NOTE]
+> JS initializers for Blazor Web Apps is an upcoming feature that will appear with the release of .NET 8 in mid-November.
+
+For Blazor Web Apps:
+
+The following example demonstrates JS initializers that load custom scripts before and after the Blazor Web App has started by appending them to the `<head>` in `beforeWebStart` and `afterWebStarted`:
+
+```javascript
+export function beforeWebStart() {
+  var customScript = document.createElement('script');
+  customScript.setAttribute('src', 'beforeStartScripts.js');
+  document.head.appendChild(customScript);
+}
+
+export function afterWebStarted() {
+  var customScript = document.createElement('script');
+  customScript.setAttribute('src', 'afterStartedScripts.js');
+  document.head.appendChild(customScript);
+}
+```
+
+The preceding `beforeWebStart` example only guarantees that the custom script loads before Blazor starts. It doesn't guarantee that awaited promises in the script complete their execution before Blazor starts.
+
+For Blazor Server, Blazor WebAssembly, and Blazor Hybrid apps:
+
+:::moniker-end
+
+:::moniker range=">= aspnetcore-6.0"
+
 The following example demonstrates JS initializers that load custom scripts before and after Blazor has started by appending them to the `<head>` in `beforeStart` and `afterStarted`:
 
 ```javascript
-export function beforeStart(options, extensions) {
+export function beforeStart() {
   var customScript = document.createElement('script');
   customScript.setAttribute('src', 'beforeStartScripts.js');
   document.head.appendChild(customScript);
 }
 
-export function afterStarted(blazor) {
```

Comment:
What's the reason for removing these parameters?

Reply:
I'll reverse it ... they just weren't used in the example. Np ... I'll put them back.